test/development/app-render-error-log/app-render-error-log.test.ts

```diff
@@ -32,9 +32,9 @@ createNextDescribe(
 
       await check(() => cliOutput, /digest:/)
       expect(cliOutput).toInclude('Error: boom')
-      expect(cliOutput).toInclude('at fn2 (./app/fn.ts:6:11)')
-      expect(cliOutput).toInclude('at fn1 (./app/fn.ts:9:5')
-      expect(cliOutput).toInclude('at EdgePage (./app/edge/page.tsx:12:45)')
+      expect(cliOutput).toInclude('at fn2 (./app/fn.ts')
+      expect(cliOutput).toInclude('at fn1 (./app/fn.ts')
+      expect(cliOutput).toInclude('at EdgePage (./app/edge/page.tsx')
```

Comment:
The line numbers here were not correct in the first place - I assume that it has something to do with our SWC generated code / sourcemaps.